docs/handling-opentelemetry-references.md

```diff
@@ -0,0 +1,66 @@
+# Handling OpenTelemetry SDK/Instrumentation References in Applications with Automatic Instrumentation
+
+## Summary
```

Comment:
We probably don't need to label this top section as Summary, but instead just have the paragraph on its own.